src/util/ApiClient.ts

```diff
@@ -73,12 +69,13 @@ class ApiClient {
                 return await response.text();
             }
             error = {
+                name: 'Failed to fetch',
                 errorCode: response.status,
                 message: response.statusText,
             };
         } catch (err) {
-            console.log(err); // tslint:disable-line no-console
             error = {
+                name: 'Connectivity error',
```

Comment:
Name field is only being used within tests (if i'm not blind) so why add name property? In tests, you can just compare error message

Reply:
Name is a default property of the Typescript Error interface